code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm

```diff
@@ -194,3 +194,76 @@
 	if(!can_use_ability(target, override_flags = XACT_IGNORE_SELECTED_ABILITY))
 		return ..()
 	return TRUE
+
+// ***************************************
+// *********** Evasion
+// ***************************************
+/datum/action/xeno_action/evasion
+	name = "Evasion"
+	action_icon_state = "unburrow"
+	mechanics_text = "Take evasive action, forcing non-friendly projectiles that would hit you to miss so long as you keep moving."
+	plasma_cost = 20
+	cooldown_timer = 30 SECONDS
+	keybind_signal = COMSIG_XENOABILITY_EVASION
+
+
+/datum/action/xeno_action/evasion/action_activate()
+	var/mob/living/carbon/xenomorph/runner/R = owner
+
+	R.do_jitter_animation(2000)
+	R.visible_message("<span class='warning'>[R.name] begins to move erratically!</span>", \
+	"<span class='xenodanger'>We move erratically, making us impossible to hit with projectiles; the next [RUNNER_EVASION_STACKS] projectiles that would hit us will now miss.</span>")
+
+	addtimer(CALLBACK(GLOBAL_PROC, .proc/to_chat, R, "<span class='highdanger'>We begin to slow as we tire.</span>"), RUNNER_EVASION_DURATION * 0.7)
+	addtimer(CALLBACK(R, /mob/.proc/playsound_local, R, 'sound/voice/hiss4.ogg', 50), RUNNER_EVASION_DURATION * 0.7)
```

Comment:
put these two into a proc on this ability

Reply:
Okay.